docs/robotframework.rst

```diff
@@ -221,18 +222,61 @@ decorator provided by CumulusCI. Unlike traditional Robot Framework
 keyword libraries, you may define multiple sets of keywords in a
 single file.
 
-For example, the following example shows the definition of a page
-object for the listing page of the Contact object. It adds a new
-keyword named :code:`Click contact link`:
+When you create a page object class, you should start by inheriting
+from one of the following base classes provided by cumulusci. No
+matter which class your inherit from, your class gets the following
+predefined properties:
+
+- **self.object_name** - the name of the object related to the
+  class. This is defined via the `object_name` parameter to the
+  ``pageobject`` decorator.
```

Comment:
Is it worth mentioning how this will add the namespace prefix when needed?

Reply:
Yes, good catch. Thanks!